contracts/src/rollup/RollupCore.sol

```diff
@@ -31,11 +31,16 @@ abstract contract RollupCore is IRollupCore, PausableUpgradeable {
     uint256 public baseStake;
     bytes32 public wasmModuleRoot;
 
-    IBridge public delayedBridge;
-    ISequencerInbox public sequencerBridge;
+    IBridge public bridge;
     IOutbox public outbox;
-    IRollupEventBridge public rollupEventBridge;
+    ISequencerInbox public sequencerInbox;
+    IRollupEventInbox public rollupEventInbox;
     IChallengeManager public override challengeManager;
+
+    // misc useful contracts when interacting with the rollup
+    address public validatorUtils;
+    address public validatorWalletCreator;
```

Comment:
Let's add the delayed inbox to this list too. I know in theory you could iterate over the bridge's allowed inboxes but that's fragile if e.g. we decide to add more inboxes in the future.